src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -262,7 +262,7 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
     memberBlock ^^ ObjectType
 
   lazy val memberBlock: Parser[List[MemberTree]] =
-    "{" ~> rep(typeMember <~ opt(";" | ",")) <~ "}"
+    "{" ~> rep((typeMember | privateMember) <~ opt(";" | ",")) <~ "}"
```

Comment:
Er, sorry, I meant as an alternative in the definition of `lazy val typeMember`. So a `typeMember` can be a `callMember`, etc., or a `privateMember`. `typeMember` is any kind of member than kind appear in a `memberBlock`.

Reply:
Ah sure, totally makes sense looking more closely. So next try :)